docs/en/workshop/a-molecular-unfolding/build-model-detail.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Build Model (Technical Detail)
+weight: 10
+---
+
+The following content is the technical details for building model
+
+### Problem Definition
+
+In this problem, the ligand is considered as a flexible set of atoms. Strictly speaking, 
+it can be seen as a set of chemical bonds (edges). These bonds have fixed length and 
+only a subset of them are rotatable. Because there are some rotatable bonds (torsions)
+, the molecule is split into different disjointed fragments. Take one bond for instance, 
+the rightmost rotatable one, it splits the molecule into the left and right fragments. 
+These fragments can rotate independently from each other around the axis of the bond. This 
+idea is graphically reported in the following figure. 
+
+<center>
+
+ ![Rotatable Bonds](../../images/rotatable-bonds.png)
+ Figure 1: Rotatable Bonds [<sup>1</sup>](#qmu-paper)
+ 
+ </center>
+
+ As it indicates, the objective of MU is to find the shape of the ligand that can maximizes 
```

Comment:
don't use `MU` abbreviation in first occurrence

Reply:
described in the former page:
...
In this work, actually the first phase, ligand expansion or the molecular unfolding (MU), is focused and 
implemented using quantum annealer
...